LanguageFeatures/Augmentation-libraries/augmented_expression_A01_t03_lib.dart

```diff
@@ -23,6 +23,12 @@ augment String get topLevelGetter {
 //^^^^^^^^^
 // [analyzer] unspecified
 // [cfe] unspecified
+  var f = () {
+    augmented();
+//  ^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  }
```

Comment:
We'd need this in several locations:
```suggestion
  };
```